lib/aurora_bootstrapper/notifier.rb

```diff
@@ -12,7 +12,48 @@ def export_date
     
     # ENV is string to string dictionary
     def export_date_override
-      DateTime.now.strftime("%Y-%m-%d") if ENV.key?('EXPORT_DATE_OVERRIDE')
+      date_override = nil
+
+      if ENV.key?('EXPORT_DATE_OVERRIDE')
+        now = Date.today
+
+        # expiration time is 30 days
+        for i in 1..30
```

Comment:
If we check today's folder, there would be two cases (1) no DONE file (2) with DONE file. For (1) we could safely skip the check; for (2) it could mean the job somehow gets restarted even after the previous run is successful. This should never happen as our job is set to be restarted only on failure.

Reply:
makes sense!